partiql-cli/Cargo.toml

```diff
@@ -41,14 +41,45 @@ supports-unicode = "1.0.2"
 supports-hyperlinks = "1.2.0"
 termbg = "0.4.1"
 shellexpand = "2.1.0"
-partiql-parser = { path = "../partiql-parser" }
-partiql-source-map = { path = "../partiql-source-map" }
-partiql-ast = { path = "../partiql-ast" }
 
 
 thiserror = "1.0.31"
 miette = { version ="4.7.1", features = ["fancy"] }
+clap = { version = "3.2.13", features = ["derive"] }
+
+# serde
+serde = { version ="1.*", features = ["derive"], optional = true }
+serde_json = { version ="1.*", optional = true }
 
+### Dependencies for the `render` feature
+viuer = { version ="0.6.1", features = ["sixel"], optional = true }
+image = { version ="0.24.2", optional = true }
+graphviz-sys = { version ="0.1.3", optional = true }
```

Comment:
FYI, if you're running this on macOS and run into the following error:

```
❯ cargo run --release --features visualize -- ast -Tdisplay "SELECT num.*, num BETWEEN 2 AND 15 FROM (SELECT tbl.a[8] + ((4/-2)+(7*sin(7))-2.0) - xform(1,position('x' in 'yxz')), NULL FROM tbl WHERE tbl.b LIKE 'foo%') as num"
   Compiling graphviz-sys v0.1.3
   Compiling resvg v0.23.0
   Compiling partiql-parser v0.0.0 (/Users/caialan/wksp/partiql-lang-rust/partiql-parser)
   Compiling viuer v0.6.1
   Compiling termbg v0.4.1
The following warnings were emitted during compilation:

warning: couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
warning: set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

error: failed to run custom build command for `graphviz-sys v0.1.3`

Caused by:
  process didn't exit successfully: `/Users/caialan/wksp/partiql-lang-rust/partiql-cli/target/release/build/graphviz-sys-241bfa87d370ff4f/build-script-build` (exit status: 101)
  --- stdout
  cargo:rustc-link-lib=gvc
  cargo:rustc-link-lib=cgraph
  cargo:rustc-link-lib=pathplan
  cargo:rustc-link-lib=cdt
  cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
  cargo:warning=set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

  --- stderr
  ./wrapper.h:1:10: fatal error: 'graphviz/gvc.h' file not found
  ./wrapper.h:1:10: fatal error: 'graphviz/gvc.h' file not found, err: true
  thread 'main' panicked at 'Unable to generate bindings: ()', /Users/caialan/.cargo/registry/src/github.com-1ecc6299db9ec823/graphviz-sys-0.1.3/build.rs:43:14
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

You may also need to install `graphviz` (i.e. `brew install graphviz`). Credit to Josh for the fix.

Reply:
Could add the `brew install graphviz` as a step in the README to run locally. Also may need to test on other OS